src/main/java/org/opentripplanner/api/common/RoutingResource.java

```diff
@@ -564,6 +565,15 @@
     @QueryParam("priceCostWeight")
     private Double priceCostWeight;
 
+    @QueryParam("walkPrice")
+    private Double walkPrice;
```

Comment:
I am a bit worried about this `walkPrice` as a way of forcing us to use other vehicles (I think we already have `walkReluctance` which handles that case), but lets see how it will influence responses.